packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -89,6 +89,7 @@ export const RecipientAvatar = ({ recipient, ...rest }) => {
       image={`${client.options.uri}${recipient.avatarPath}`}
       text={getInitials(recipient)}
       textId={getDisplayName(recipient)}
+      disabled={recipient.status === 'pending'}
```

Comment:
Je viens de pousser une PR pour mettre les valeurs possibles de status, mais dans ce cas il faudrait je pense tester le pending et mail-not-send (https://github.com/cozy/cozy-doctypes/pull/152)